### ISSUE TITLE
vagrant 2.0.1 and 2.0.2 install old version of plugin

### ISSUE THREAD

**Issue by jennyowen** [hashicorp/vagrant#9548]
### Vagrant version
Vagrant 2.0.2
Vagrant 2.0.1

### Host operating system
Ubuntu 16.04

### Expected behavior
version 1.0.3 of `vagrant-cucumber` plugin should be installed. This is what happens with Vagrant 2.0.0.

### Actual behavior
version 0.0.9 of `vagrant-cucumber` plugin is installed.

### Steps to reproduce
1. install vagrant 2.0.2 or 2.0.1
2. `vagrant plugin install vagrant-cucumber`

I'm not sure if vagrant is correctly referencing the latest https://rubygems.org. If I query the gem repository from command line, the correct version (1.0.3) is listed.
`gem list --remote vagrant-cucumber`

I can ping rubygems.org without any problems.

### References
This may be related to issue https://github.com/hashicorp/vagrant/issues/9516, although I do not see any error messages.

**Comment by briancain** [comment]
Hello! Thanks for opening an issue. When opening a new issue on the Vagrant project, we ask that everyone provide a [gist](https://gist.github.com/) containing the behavior you're seeing with the `--debug` flag included. This helps us figure out what's actually going on.

I just tried to reproduce this with Vagrant 2.0.2 and I got the expected version installed:

```
vagrant@vagrant:~$ vagrant plugin install vagrant-cucumber
Installing the 'vagrant-cucumber' plugin. This can take a few minutes...
Fetching: open4-1.3.4.gem (100%)
Fetching: Platform-0.4.0.gem (100%)
Fetching: popen4-0.1.2.gem (100%)
Fetching: sahara-0.0.17.gem (100%)
Fetching: to_regexp-0.2.1.gem (100%)
Fetching: cucumber-wire-0.0.1.gem (100%)
Fetching: multi_test-0.1.2.gem (100%)
Fetching: gherkin-4.1.3.gem (100%)
Fetching: diff-lcs-1.3.gem (100%)
Fetching: cucumber-core-1.5.0.gem (100%)
Fetching: cucumber-2.99.0.gem (100%)
Fetching: vagrant-cucumber-1.0.3.gem (100%)
Installed the plugin 'vagrant-cucumber (1.0.3)'!
vagrant@vagrant:~$ vagrant --version
Vagrant 2.0.2
```

Thanks!

**Comment by jennyowen** [comment]
I reinstalled Vagrant 2.0.2 and was able to reproduce the problem. Here is the gist:
https://gist.github.com/jennyowen/47a98016d62007737c6703697d0de121#file-hashicorp-vagrant-issue-9548

I was installing 2.0.2 on top of my existing 2.0.0 installation, and I thought perhaps that could be related to the problem. So to get a cleaner reproduction I uninstalled vagrant 2.0.0 and deleted my entire `~/.vagrant.d` folder. I then installed vagrant 2.0.2 and the plugin. The correct version of the plugin was installed the second time.

I went back and tried reproducing what I did originally (i.e. installing vagrant 2.0.2 on top of 2.0.0 with existing plugins), so that I could get you a better gist. I wasn't able to reproduce the problem again though.
